arches/app/views/api/__init__.py

```diff
@@ -2383,3 +2383,128 @@ def delete(self, request, identifier=None):
                 status=400,
             )
         return JSONResponse(status=204)
+
+
+class UserPreference(APIBase):
```

Comment:
I would get the word view in this name somehow. See next comment, maybe this becomes UserPreferenceListView.